copylot/hardware/orca_camera/camera.py

```diff
@@ -1,5 +1,40 @@
 from copylot.hardware.orca_camera.dcam import Dcamapi, Dcam
 
+import cv2
+import numpy
+from copylot.hardware.orca_camera.dcamapi4 import DCAM_IDPROP
+
+
+def dcamtest_show_framedata(data, windowtitle, iShown):
```

Comment:
This looks like a nice demo function but I think living in the wrong place. Could you move this to a suitable demo file @xiyuyi ?